waiter/src/waiter/scheduler/cook.clj

```diff
@@ -471,16 +474,18 @@
       (mesos/retrieve-directory-content-from-host cook-api host log-directory)))
 
   (service-id->state [_ service-id]
-    {:failed-instances (service-id->failed-instances service-id->failed-instances-transient-store service-id)})
+    (-> (retrieve-syncer-state-fn service-id)
```

Comment:
Should the syncer state be in a key like `:syncer` instead of the top level map?

Reply:
Eagerly made the change, If you disagree, will revert.

Reply:
@sradack ready for review